config.json

```diff
@@ -313,6 +313,18 @@
         "strings",
         "type_conversion"
       ]
+    },
+    {
+      "slug": "isbn-verifier",
+      "uuid": "c52dfe2b-5634-4c0c-8ea2-3ca86e5d9a33",
+      "core": false,
```

Comment:
I think this would be a good core exercise. Otherwise, I think this PR is ready.

Reply:
@Stargator Sounds good, where did you want to have this exercise take place in this list of core exercises? Add to the end?

Reply:
It can stay where it is.